transaction-status/src/parse_token/extension/confidential_transfer_fee.rs

```diff
@@ -43,14 +43,34 @@ pub(in crate::parse_token) fn parse_confidential_transfer_fee_instruction(
             let mut value = json!({
```

Comment:
I can't comment on unchanged changes, but the `InitializeConfidentialTransferFeeConfig` might be missing the instruction data in json?

Reply:
Done in 0d50498a99